bin/node-template/src/service.rs

```diff
@@ -160,7 +160,6 @@ pub fn new_full<C: Send + Default + 'static>(config: Configuration<C, GenesisCon
 				grandpa_link,
 				service.network(),
 				service.on_exit(),
-				service.spawn_task_handle(),
```

Comment:
None of `finality-grandpa` nor `network-gossip` need an executor to spawn tasks after this patch. Thus the logic passing said executor to those crates is removed in `bin/node`, `bin/node-template`, `client/service` and a lot of tests.